packages/slate/src/controllers/change.js

```diff
@@ -332,20 +332,20 @@ function getDirtyPaths(operation) {
     case 'insert_node': {
```

Comment:
I think we need to use `...ancestors` for the first switch block too, to allow for `marks` rules to trigger on higher-up blocks.

Reply:
Ahh good call, that works for me locally 👍

Reply:
This is probably good. I'm seeing two bugs, but it's more likely than not that this is caused by changes to my schema rules from before I realized the problem wasn't with my rules.